wemake_python_styleguide/violations/complexity.py

```diff
@@ -574,3 +576,31 @@ class TooManyForsInComprehensionViolation(ASTViolation):
     should_use_text = False
     error_template = 'Found a comprehension with too many `for` statements'
     code = 224
+
+
+@final
+class TooManyContextManagerAssignmentsViolation(ASTViolation):
+    """
+    Forbid multiple assignment targets for context managers.
+
+    Reasoning:
+        Following "as" with another context manager looks like a tuple.
+        Emitted when a with statement component returns multiple values
+        and uses name binding with as only for a part of those values,
+        as in with ctx() as a, b.This can be misleading, since it's not
+        clear if the context manager returns a tupleor if the node
+        without a name binding is another context manager.
+
+    Example::
+
+        # Wrong:
+            with open('') as first, second:
```

Comment:
Please, fix the indentation. `# Wrong:` and `with open('')` should have the same indentation.